FPGA github workflow is failing

Hello,

It seems that the [FPGA Github action](https://github.com/TinyTapeout/ttsky-verilog-template/blob/main/.github/workflows/fpga.yaml) is failing because of a [missing `--create-fpga-bitstream` flag](https://github.com/search?q=repo%3ATinyTapeout%2Ftt-support-tools%20--create-fpga-bitstream&type=code) in tt_tool.py in [TinyTapeout/tt-support-tools](https://github.com/search?q=repo%3ATinyTapeout%2Ftt-support-tools).

It results in the following message:

```
Run python tt/tt_tool.py --create-fpga-bitstream
usage: tt_tool.py [-h] [--project-dir PROJECT_DIR] [--yaml YAML] [--debug]
                  [--ihp] [--gf] [--print-cell-summary]
                  [--print-cell-category] [--print-stats] [--print-warnings]
                  [--print-wokwi-id] [--print-top-module] [--check-docs]
                  [--create-pdf] [--template-version [TEMPLATE_VERSION]]
                  [--create-svg] [--create-png] [--no-docker]
                  [--create-user-config] [--harden] [--create-tt-submission]
                  [--open-in-klayout] [--open-in-openroad]
tt_tool.py: error: unrecognized arguments: --create-fpga-bitstream
Error: Process completed with exit code 2.
```

Is there anything that I am missing?

Thanks

--- a/fpga/ice40up5k/action.yml
+++ b/fpga/ice40up5k/action.yml
@@ -54,7 +54,7 @@ runs:
     - name: Create the bitstream
       shell: bash
       run: |
-        python tt/tt_tool.py --create-fpga-bitstream
+        python tt/tt_fpga.py harden
 
     - name: Upload the bitstream
       if: always()

